### DISCUSSION TITLE
Need for MAR linear configuration

### DISCUSSION THREAD

**Discussion by TerrencePai** [open-mmlab/mmselfsup#684] [Q&A]
Can anybody provide the linear detection configuration file of MAE? The top1 accuracy obtained by using the default linear detection configuration file of mmselfsup is only 59.236%, which is far from the result in the paper.

**Comment by YuanLiuuuuuu** [comment]
Do you mean linear classification? If that is the case, please use this [file](https://github.com/open-mmlab/mmselfsup/blob/dev-1.x/configs/benchmarks/classification/imagenet/vit-base-p16_linear-8xb2048-coslr-90e_in1k.py) and the detailed results are reported [here](https://github.com/open-mmlab/mmselfsup/tree/dev-1.x/configs/selfsup/mae)
